ts/src/coinlist.ts

```diff
@@ -0,0 +1,2246 @@
+        //         "epoch_timestamp":"2023-10-26T08:20:56.307Z",
+        //         "origin":"web",
+        //         "self_trade_prevention":null,
+        //         "client_id":null,
+        //         "created_at":"2023-10-26T08:20:56.307Z",
+        //         "symbol":"BTC-USDT",
+        //         "size":"0.0003",
+        //         "side":"sell",
+        //         "type":"limit",
+        //         "post_only":false,
+        //         "size_filled":"0.0000"
+        //     }
+        //
+        // createOrder
+        //     {
+        //         "symbol": "BTC-USDT",
+        //         "type": "market",
+        //         "side": "sell",
+        //         "size": "0.0003",
+        //         "order_id": "cad67c0f-9aec-4ac8-ac03-aaf5db299ff7",
+        //         "trader_id": "9c6f737e-a829-4843-87b1-b1ce86f2853b"
+        //     },
+        //
+        // cancelOrder
+        //     {
+        //         message: 'Cancel order request received.',
+        //         order_id: 'd36e7588-6525-485c-b768-8ad8b3f745f9',
+        //         timestamp: '2023-10-26T14:36:37.559Z'
+        //     }
+        //
+        // cancelOrders
+        //     {
+        //         message: 'Order cancellation request received.',
+        //         timestamp: '2023-10-26T10:29:28.652Z'
+        //     }
+        //
+        // cancelAllOrders
+        //     {
+        //         message: 'Order cancellation request received.',
+        //         timestamp: '2023-10-26T10:29:28.652Z'
+        //     }
+        //
+        const id = this.safeString (order, 'order_id');
+        const marketId = this.safeString (order, 'symbol');
+        market = this.safeMarket (marketId, market);
+        const clientOrderId = this.safeString (order, 'client_id', undefined);
+        let timestampString = this.safeString2 (order, 'created_at', 'epoch_timestamp', undefined);
+        if (timestampString === undefined) {
+            timestampString = this.safeString (order, 'timestamp', undefined);
+        }
+        const timestamp = this.parse8601 (timestampString);
+        const status = this.parseOrderStatus (this.safeString (order, 'status'));
+        const type = this.parseOrderType (this.safeString (order, 'type'));
+        const side = this.safeString (order, 'side');
+        const price = this.safeString (order, 'price', undefined);
+        const stopPrice = this.safeString (order, 'stop_price', undefined);
+        const average = this.safeString (order, 'average_fill_price', undefined); // from documentation
+        const amount = this.safeString (order, 'size');
+        const filled = this.safeString (order, 'size_filled');
+        const feeCost = this.safeString (order, 'fill_fees');
+        const postOnly = this.safeValue (order, 'post_only', undefined);
+        let fee = undefined;
+        if (feeCost !== undefined) {
+            fee = {
+                'currency': 'USD', // todo: or undefined or quote?
```

Comment:
When selling I think you will pay the fees in the asset you just sold, so we can check the side to decide